metals/src/main/scala/scala/meta/internal/metals/watcher/FileWatcher.scala

```diff
@@ -146,9 +146,9 @@ object FileWatcher {
       // and run some tests with `-Dswoval.log.level=debug` to see which files are registered
       // and which file events are received.
       val directoriesToCreate =
-        filesToWatch.sourceDirectories ++ filesToWatch.sourceFiles.map(
-          _.getParent()
-        )
+        filesToWatch.sourceDirectories ++
```

Comment:
This seems to break semanticdb on my machine altogether :thinking:

Reply:
That's a bummer :/